docs/excel/excel-add-ins-multiple-ranges.md

```diff
@@ -0,0 +1,208 @@
+ 
+## Creating RangeAreas and setting properties
+
+You can create a first `RangeAreas` object in two ways:
+
+- Call `Worksheet.getRanges()` and pass it a string with comma-delimited range addresses. (If any range you want to include has been made into a [NamedItem](https://docs.microsoft.com/javascript/api/excel/excel.nameditem), you can include the name, instead of the address, in the string.)
+- Call `Workbook.getSelectedRanges()`. This method returns a `RangeAreas` representing all the ranges that are selected when it runs.
+
+Once you have a `RangeAreas` object, you can create others using the methods on the object that return `RangeAreas` such as `getOffsetRangeAreas` and `getIntersection`.
+
+> [!NOTE]
+> You cannot directly add additional ranges to a `RangeAreas` object. For example, the collection in `RangeAreas.areas` does not have an `add` method. It is possible to push an additional `Range` object onto the `RangeAreas.areas.items` array, but this is a bad practice because `RangeAreas` properties and methods behave as if it weren't there. For example, the `areaCount` property does include ranges pushed in this way, and the `RangeAreas.getItemAt(index)` throws an error if `index` is larger than the `areasCount-1`. Similarly, it is a bad practice to delete a `Range` object in the `RangeAreas.areas.items` array by getting a reference to it and calling its `Range.delete` method. Although the `Range` object is deleted, the properties and methods behave, or try to, as if it is still in existence.
+
+Setting a property on a `RangeAreas` has the effect of setting the corresponding property on all the ranges in the `RangeAreas.areas` collection.  
+
+The following is an example of setting a property on multiple ranges. Note the function is intended to highlight the ranges **F3:F5** and **H3:H5** (and no others).
+
+```js
+Excel.run(function (context) {
+    const sheet = context.workbook.worksheets.getActiveWorksheet();
+    const rangeAreas = sheet.getRanges("F3:F5, H3:H5");
+    rangeAreas.format.fill.color = "pink";
+
+    return context.sync();
+})
+```
+
+This example applies to scenarios in which you can hard code the range addresses that you pass to `getRanges` or easily calculate them at runtime. This would include the following scenarios, among others:
+
+- The code runs in the context of a known template.
+- The code runs in the context of imported data where the schema of the data is known.
+
+When you can't know at coding-time which ranges you need to operate on, you must discover them at runtime. The next section discusses these scenarios. 
+
+### Programmatically discovering range areas
+
+The `Range.getSpecialCells()` and `Range.getSpecialCellsOrNullObject()` methods enable you to find at runtime the ranges that you want to operate on based on the characteristics of the cells and the type of the values of the cells. The following is an example. About this code, note:
```

Comment:
I'd add these method signatures somewhere. Later on you're referencing the parameters in an order (i.e. the second parameter) and it'd be nice to see that upfront.